docs/basic-features/script.md

```diff
@@ -118,7 +118,7 @@ export default function Home() {
         id="stripe-js"
         src="https://js.stripe.com/v3/"
         onLoad={() => {
-          this.setState({ stripe: window.Stripe('pk_test_12345') })
+          setState({ stripe: window.Stripe('pk_test_12345') })
```

Comment:
This is missing a `useState` hook

Reply:
Thanks for the answer, I asked about it in the description:
> Let me know if you'd like a useState for the sake of a complete example.

Reply:
```suggestion
          setStripe({ stripe: window.Stripe('pk_test_12345') })
```